pkgs/development/python-modules/sqlalchemy-utils/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, isPy3k, fetchPypi, buildPythonPackage
+, six, sqlalchemy
+, pytz, mock, pytest, flexmock, python-dateutil }:
+
+buildPythonPackage rec {
+  pname = "sqlalchemy-utils";
+  version = "0.33.6";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "SQLAlchemy-Utils";
+    sha256 = "45ab41c90bfb8dd676e83179be3088b3f2d64b613e3b590187163dd941c22d4c";
+  };
+
+  propagatedBuildInputs = [
+    six
+    sqlalchemy
+  ];
+
+  # TypeError: sort_query() missing 1 required positional argument: 'query'
+  doCheck = !isPy3k;
+  checkInputs = [
+    pytz
+    mock
+    pytest
+    flexmock
+    python-dateutil
+  ];
+
+  meta = with lib; {
+    homepage = https://github.com/kvesteri/sqlalchemy-utils;
+    description = "Various utility functions and datatypes for SQLAlchemy";
+    longDescription = ''
+      SQLAlchemy-Utils provides custom data types and various utility functions
+      for SQLAlchemy.
```

Comment:
There's really no point in having `longDescription` if all it does is repeating `description`.